pydis_site/apps/content/README.md

```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
```

Comment:
All references to "Markdown" should be capitalised.

Reply:
ca2c583b15d46d7821a03ec047678429f96c7e03